server/services/team/service.ts

```diff
@@ -839,6 +884,6 @@ export class TeamService extends ServiceClass implements ITeamService {
 			},
 		}).toArray();
 
-		return rooms;
+		return rooms as IRoom[]; // TODO: Remove
```

Comment:
is this really needed?
```suggestion
		return rooms;
```